src/goDeclaration.ts

```diff
@@ -202,7 +202,15 @@ function definitionLocation_gogetdoc(input: GoDefinitionInput, token: vscode.Can
 				}
 				let goGetDocOutput = <GoGetDocOuput>JSON.parse(stdout.toString());
 				let match = /(.*):(\d+):(\d+)/.exec(goGetDocOutput.pos);
-				let definitionInfo = {
+				let definitionInfo: {
```

Comment:
>definitionInfo [](start = 8, length = 14)

from the containing functions, isn't this type supposed to be :`GoDefinitionInformation` ?

Reply:
same for the other ones in this file

---
In reply to: [253677626](https://github.com/Microsoft/vscode-go/pull/2294#discussion_r253677626) [](ancestors = 253677626)